### DISCUSSION TITLE
How to know what version of Maui/.NET each pull request goes into

### DISCUSSION THREAD

**Discussion by jonmdev** [dotnet/maui#22625] [Q&A]
### Description

I am interested in knowing what version of Maui features each fix that is pushed to the repository. How can one know this?

Is there any way to browse the Maui repository at each identified version release easily so one can see if a fix went into it or compare the dates of the merge?

For example, I am highly interested in this fix to see if it improves my custom "collection view" type element's performance:

https://github.com/dotnet/maui/pull/21291

But I am unclear what version of Maui/.NET features that update.

Any help?

### Public API Changes

Would be nice if each merge had the version of Maui/.NET that it has gone into or if there is some easy way to verify this.

### Intended Use-Case

So we can see if the various fixes are improving the issues or we need to perform other tests to find other targets.

**Comment by samhouts** [comment]
For pull requests with linked issues, you can check back to the linked issues to see if there are "fixed-in" labels. 

For example, https://github.com/dotnet/maui/issues/18505 has 
- [fixed-in-8.0.20](https://github.com/dotnet/maui/labels/fixed-in-8.0.20)
- [fixed-in-8.0.40](https://github.com/dotnet/maui/labels/fixed-in-8.0.40)
- [fixed-in-9.0.0-preview.3.10457](https://github.com/dotnet/maui/labels/fixed-in-9.0.0-preview.3.10457)

I try to add these soon after the releases are out, but I do get behind a bit. :)

**Comment by stefandb1995** [comment]
You could also open the releases page (https://github.com/dotnet/maui/releases) and search for the ticket name. Looks like this was first added in [8.0.20 SR4](https://github.com/dotnet/maui/releases/tag/8.0.20).
